Assign analyst to script bug (#8447)
demisto/content: Packs/CommonScripts/Scripts/script-AssignAnalystToIncident_README.md

 * online: The analyst is picked randomly from all online analysts, according to the provided roles (if no roles are provided, this will fetch all users).
 * current: The user that executed the command.

> When the chosen  `assignBy` argument is on of: machine-learning, top-user or  less-busy-user, 

Reviewer: ```suggestion
When the chosen  `assignBy` argument is either: machine-learning, top-user or  less-busy-user, 
```
